pkg/operatingsystem/cos/git-reader.go

```diff
@@ -0,0 +1,115 @@
+package cos
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+)
+
+const (
+	// MilestoneMin is the lowest active milestone. Lower value milestones are not built. See
+	// https://cloud.google.com/container-optimized-os/docs/concepts/versioning.
+	MilestoneMin = 89
```

Comment:
I'll have another look around, only found https://cloud.google.com/container-optimized-os/docs/concepts/versioning, but it may be the data comes from an API. I'm not over-eager to scrape the HTML for this though.

Current way of hardcoding means that occasionally we would have to bump this, the failure mode in the meantime would be building for a deprecated milestone which isn't too much of a problem.

Reply:
Let's just hard code it then :)

Reply:
😆 perhaps if it's a real pain, thanks!

Reply:
Ah unfortunately the data is embedded in that page:

![image](https://user-images.githubusercontent.com/665830/203962263-ecc0ceac-f767-4401-a693-bfc1a4af12e4.png)

There is also the [gcloud compute images list api](https://cloud.google.com/sdk/gcloud/reference/compute/images/list) but that requires auth :/